core/src/test/java/com/sap/ai/sdk/core/client/DeploymentUnitTest.java

```diff
@@ -62,8 +62,7 @@ void getDeployments() {
                               "ttl": null,
                               "details": {
                                 "scaling": {
-                                  "backendDetails": {},
-                                  "backend_details": {}
+                                  "backendDetails": {}
```

Comment:
```suggestion
                                  "backendDetails": {},
                                  "backend_details": {}
```
Why change the response payload?

Reply:
currently the server still sends both, and I figured it's helpful to have the payload here not diverge from actual server responses. But it's not really important since we ignore the old variant.